src/commands/tokens/list.ts

```diff
@@ -0,0 +1,24 @@
+import { Command, CommandArgs, CommandResult, help, success } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { reportTokenInfo } from "./lib/format-token";
+import { MobileCenterClient, models, clientRequest } from "../../util/apis";
+
+const debug = require("debug")("mobile-center-cli:commands:apps:list");
+import { inspect } from "util";
+
+@help("Get list of configured applications")
+export default class ApiTokenListCommand extends Command {
+  constructor(args: CommandArgs) {
+    super(args);
+  }
+
+
+  async run(client: MobileCenterClient): Promise<CommandResult> {
+    const apiTokensResponse = await out.progress("Getting API tokens ...",
+      clientRequest<models.ApiTokenResponse[]>(cb => client.account.getApiTokens(cb)));
+
+    apiTokensResponse.result.map(apiToken => reportTokenInfo(apiToken));
```

Comment:
For future consideration - use out.table to display this instead of out.report. Or use out.report but pass the entire array in one shot instead of the external map call.